base/int.jl

```diff
@@ -2,23 +2,25 @@
 
 ## integer arithmetic ##
 
-const IntTypes = (Int8, UInt8, Int16, UInt16, Int32, UInt32,
-                  Int64, UInt64, Int128, UInt128)
+const Signed64Types    = (Int8,Int16,Int32,Int64)
```

Comment:
The 64 types are used for more things than just that, for example, with the promotion rules, for example, and outside `int.jl`, in `hashing2.jl`.
I can add a comment though that, in part, they are needed for that reason.